shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -362,6 +362,7 @@ public static final class InvalidEventNumber implements Reply, WireCommand {
         final WireCommandType type = WireCommandType.INVALID_EVENT_NUMBER;
         final UUID writerId;
         final long eventNumber;
+        final long requestId;
```

Comment:
This is not backward compatible. New fields need to go at the end. So the event number field should be renamed `requestId` and a new `eventNumber` field should be added to the end.